### PR TITLE
complete

### PR DESCRIPTION
**Pull request by Khizarkhan07**

### CODE REVIEW

File: src/components/ButtonWithIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    width: '100%'
+  }
+
+  return (
+    <button
+      style={styles}
+      onClick={callback}
+      className="btn btn-light btn-outline mb-1"
+    >
+      {icon && <i className={icon + " mr-1"} style={{ color: "#fffff" }}/>}
```
Comment (ArslanHayat12) [inline src/components/ButtonWithIcon.tsx]: why inline styling?

File: src/components/column/Column.tsx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                    callback={(e) => {
+                      setTitleToggle(true);
+                    }}
+                />
+              </span>
+                      </TitleWrapper>
+                  ) : (
+                      <TextAreaWrapper>
+                        <i
+                            onClick={(e) => {
```
Comment (ArslanHayat12) [inline src/components/column/Column.tsx]: Why this function is inline why we dont usecall back

File: src/components/checkboxWithLabel.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+type props = {
+    labelOn: string,
+    labelOff: string
+}
+export class CheckboxWithLabel extends React.Component<{
```
Comment (ArslanHayat12) [inline src/components/checkboxWithLabel.tsx]: Why we have used class components 
change the into functional

Comment (Khizarkhan07) [reply src/components/checkboxWithLabel.tsx:7]: Changed at every instance.

File: src/components/ButtonWithIcon.tsx (hunk 1 above)
Comment (Khizarkhan07) [reply src/components/ButtonWithIcon.tsx]: Changed.

File: src/components/column/Column.tsx (hunk 2 above)
Comment (Khizarkhan07) [reply src/components/column/Column.tsx]: Changed at every instance in the app.

File: src/components/checkboxWithLabel.tsx (hunk 3 above)
Comment (Khizarkhan07) [reply src/components/checkboxWithLabel.tsx]: Removed the component (it was not used).

**Pull request merged**